src/Controller/Admin/SiteSettingsController.php

```diff
@@ -46,6 +46,5 @@ public function save(Request $request)
             $entry->setValue($request->request->get('value')[$key]);
             $entityManager->flush();
         }
-        exit();
     }
```

Comment:
Hello @korzol 

Do you think that it is possible to return something like JSON back to the frontend? It looks kind of unfinished and I know that this passed a code review when it was originally merged couple of months ago but now we can make things better.

Reply:
Hello @ceckoslab 
I am going to create another issue hence branch and fix it there because there are few more things to fix/change